index.bs

```diff
@@ -1249,8 +1272,10 @@ The <dfn method for=FileSystemSyncAccessHandle>close()</dfn> method steps are:
 
 1. If [=this=]'s [=[[state]]=] is "`closed`", return.
 1. Set [=this=]'s [=[[state]]=] to "`closed`".
+1. Let |onLockReleasedAlgorithm| be an algorithm that does nothing.
 1. [=file entry/lock/release|Release the lock=] on
-   [=this=]'s [=FileSystemSyncAccessHandle/[[file]]=].
+   [=this=]'s [=FileSystemSyncAccessHandle/[[file]]=] with |onLockReleasedAlgorithm|.
+1. Wait for |onLockReleasedAlgorithm| to run.
```

Comment:
Now that releasing a lock is async, presumably we can't just post a task to release the lock and return. Here's my attempt at saying "wait until the lock is released to return from `close()` - let me know if there's a better way to do this. Thanks!

Reply:
I think it would be slightly clearer for _onLockReleasedAlgorithm_ to flip a boolean.

And then instead of wait you want pause as per https://html.spec.whatwg.org/multipage/webappapis.html#pause because we're inside an actual task and not in parallel.

(And also please ensure to never introduce such a thing outside of workers!)

Reply:
> I think it would be slightly clearer for _onLockReleasedAlgorithm_ to flip a boolean.

Done!

> And then instead of wait you want pause as per https://html.spec.whatwg.org/multipage/webappapis.html#pause because we're inside an actual task and not in parallel.
> 
> (And also please ensure to never introduce such a thing outside of workers!)

You betcha :)